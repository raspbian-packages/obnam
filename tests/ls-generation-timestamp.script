#!/bin/sh
# Copyright 2011  Lars Wirzenius
# 
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
# 
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU General Public License for more details.
# 
# You should have received a copy of the GNU General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

# This tests for a bug in Obnam as of 0.24.
# http://liw.fi/obnam/bugs/generation-time-stamps-wrong/

set -e

$SRCDIR/tests/backup --pretend-time='2007-08-12 01:02:03'

# Print out the generation timestamp.
$SRCDIR/tests/obnam ls | head -n1
