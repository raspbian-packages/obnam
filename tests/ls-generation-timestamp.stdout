Generation 2 (2007-08-12 01:02:03 - 2007-08-12 01:02:03)
